SoftLayer/CLI/globalip/assign.py

```diff
@@ -5,17 +5,17 @@
 
 import SoftLayer
 from SoftLayer.CLI import environment
-from SoftLayer.CLI import helpers
 
 
-@click.command()
+@click.command(epilog="More information about types and ")
```

Comment:
Did you mean to add this text? Would be a good place to mention that the user should read https://sldn.softlayer.com/reference/services/SoftLayer_Network_Subnet/route/